app/views/idv/phone_confirmation/show.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('titles.enter_2fa_code')
+
+h1.heading = t('forms.phone_confirmation.header_text')
+p Please enter the code sent to <strong>#{@unconfirmed_phone}</strong>.
+
+= form_tag([:idv_phone_confirmation], method: :put, role: 'form') do
+  = label_tag 'code', 'One-time passcode', class: 'block'
+  = number_field_tag(:code, \
```

Comment:
@brendansudol this number_field_tag needs to be wrapped in its own div like other similar forms and `align-top` on the submit button so the validation message displays correctly.